extensions/amp-bind/0.1/bind-validator.js

```diff
@@ -302,6 +302,9 @@ function createElementRules_() {
     'FIELDSET': {
       'disabled': null,
     },
+    'image': {
+      'xlink:href': null,
```

Comment:
There are security issues with `xlink:href`: https://html5sec.org/#87 

This should be:

```js
'xlink:href': {
  'allowedProtocols': {
    'http': true,
    'https': true,
  },
}
```

And we should add a unit test to bind-validator to make sure the `<image xlink:href="javascript:alert(1)">` will be disallowed.